### ISSUE TITLE
Group Editor Accessibility Options

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#187314]
Currently there are
* EditorOptions.accessibilitySupport
* EditorOptions.accessibilityPageSize
* EditorOptions.ariaLabel
* EditorOptions.screenReaderAnnounceInlineSuggestion

And now also an option for the `aria-required` property.

I think they should be grouped into `EditorOptions.accessibility`
